### PR TITLE
Dynamic channels 6

### PR DESCRIPTION
**Pull request by norkans7**
based on #1420

### CODE REVIEW

File: temba/utils/__init__.py [hunk 1]
```diff
@@ -430,4 +430,4 @@ def get_anonymous_user():
     Returns the anonymous user, originally created by django-guardian
     """
     from django.contrib.auth.models import User
-    return User.objects.get(username=settings.ANONYMOUS_USER_NAME)
```
Comment (rowanseymour) [inline temba/utils/__init__.py]: This that unrelated change again?

Comment (norkans7) [reply temba/utils/__init__.py]: my bad
